docs/guide/basic-module.md

```diff
@@ -5,52 +5,26 @@ order: 3
 # Basic Stencil Module
 
 > [!NOTE]
-> This quick start assumes you're familiar with stencil usage already. If you aren't be sure to go through the [reference documentation](/reference/) or the [quick start](/guide/).
+> This quick start assumes you're familiar with stencil usage already.
+> If you aren't be sure to go through the [reference
+> documentation](/reference/) or the [quick start](/guide/).
 
 ## Step 1: Create a module
 
-Using the `stencil create` command we're able to quickly create a module, let's start with a simple hello world module.
+Using the `stencil create` command we're able to quickly create a
+module, let's start with a simple hello world module.
 
 ```bash
 mkdir helloworld; cd helloworld
 stencil create module github.com/yourorg/helloworld
 ```
 
-You'll notice that when running that command, stencil itself was ran. This is because `stencil create` uses the [`stencil-template-base`](https://github.com/getoutreach/stencil-template-base), the [`stencil-base`](https://github.com/getoutreach/stencil-base), and the [`stencil-circleci`](https://github.com/getoutreach/stencil-circleci) modules by default. This brings automatic CI and testing support to your module.
+You'll notice this created a `stencil.yaml` file and ran `stencil` on
+itself. This is to enable a future module we'll provide that helps
```

Comment:
"This is to enable a future module we'll provide that helps  ..."

This is a confusing sentence without any more context.  I'd prolly just not talk about it yet.

Reply:
How about

> This isn't used now, but will be in the future.

I feel like we have to mention its existence since inevitably someone will ask "... why did I run this command"?

Reply:
Just lie a bit and say "this is to get the lock file going and validate that your stencil.yaml was created properly"